app/controllers/physical_server_controller.rb

```diff
@@ -33,7 +33,7 @@ def show_list
   end
 
   def textual_group_list
-    []
+    [%i(properties relationships assets)]
```

Comment:
https://github.com/ManageIQ/manageiq-ui-classic/pull/1272 can address the above. No need for a separate PR to add `[%i(properties relationships)]`.